### ISSUE TITLE
How to show more variables in browser response?

### ISSUE THREAD

**Issue by jayraydee** [marcoschwartz/aREST#114]
I am trying to add more variables via hard coding them into the firmware I am uploading via the Arduino IDE. The problem occurs when I add a third variable, they all ( temp& humidity, plus my extra) stop appearing in the browser when I navigate to cloud.arest.io/[device ID] page. I want to be able to add more variables and still have them be viewable via the webpage as I have other devices polling that page to get the variable values. 

Is there a trick I missing, or my doing it all wrong?

Thank you,
James

**Comment by marcoschwartz** [comment]
Hi James, you are doing nothing wrong, this is currently a problem with the MQTT library that I use for aREST, which limits the size of packets exchange with the server. Once this library allows larger packets the problem will be fixed.

**Comment by peeter81** [comment]
Don't know is this bug or might be my code is wrong or some library thing issue.
Besides temp & humidity variables I also added extra/custom variables into .ino file (and uploaded via Arduino IDE):
a) delay value (from delay parenthesis value) as rest variable "intervall"
for me its now delay(10000) (pause 10 sec)
+
b) rssi value (wifi signal strength, to get an idea /info for unit suitable location)

// Init variables and expose them to REST API
rest.variable("temperature",&temperature);
rest.variable("humidity",&humidity);
_rest.variable("intervall",&intervall);  //delay time in
rest.variable("rssi",&rssi);  // rssi value_

Using aREST.h in .ino file.
Everything worked fine with extra variable (intervall) was created, this and all the other variables were shown /displayed in browser.

After I added rssi variable, it was shown in browser, but value was always 0.

Is it related with limited packet size (which won't allow too "many" extra variables to create/send to browser)?
Is there any numeric restrictions (limited max. number extra variables?) that user can make "custom" (rest) variables ?

If this is limited, is it plan to extend larger packets ?
